addons/netfox/rollback/rollback-synchronizer.gd

```diff
@@ -81,8 +84,9 @@ func process_authority():
 	for property in input_properties:
 		var pe = _property_cache.get_entry(property)
 		if pe.node.is_multiplayer_authority():
-			_record_input_props.push_back(pe)
 			_auth_input_props.push_back(pe)
+		else:
+			_record_input_props.push_back(pe)
```

Comment:
This solves https://github.com/foxssake/netfox/issues/250 by making use of that variable which is unused anywhere else, yet its obvious it exists for this reason.

As in, other players must know the node types/scenes at least, so when an input comes, it can be deconstructed. This deconstruction is impossible without having the properties of the RPC'd node.

Without this PR, that variable shouldn't even exist, but it exists for this purpose as explained above: For other players to know what its properties are (since it doesnt make sense for clients to have the same scenes, yet dont know what input properties they accept)